### PR TITLE
Update project links effected due to project name change

### PR DESCRIPTION
**Pull request by AakankshaSanctis**
Changes proposed in this pull request:
-Updated the travis.yml with the latest email id of project contributors

## Notify reviewers
@achyudhk

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@AakankshaSanctis please update the README.md and etc/deploy_docs.sh files as well. @achyudhk has already made some modifications to the READNE.md. Do update the code coverage badge and cross check for the required name changes.

**Comment by AakankshaSanctis** [comment]
There is a link in the README.md for test_driver.py file (https://github.com/achyudhk/Mailing-List-Network-Analyzer/blob/development/lib/test_driver.py) which no longer  exists.@achyudhk can you look into it and put the appropriate file link.

**Comment by AakankshaSanctis** [comment]
1.  README.md
   1.  etc/deploy_docs.sh
   1.   project outline
   1.  travis.yml

have been updated

**Comment by AakankshaSanctis** [comment]
codecov and dependency badges have been added but integration needs to be done

File: README.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.org/DeveloperCAP/MLCAT.svg?branch=development)](https://travis-ci.org/DeveloperCAP/MLCAT)
 [![Maintainability](https://api.codeclimate.com/v1/badges/f1d7947c8a62338544dd/maintainability)](https://codeclimate.com/github/DeveloperCAP/MLCAT/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/f1d7947c8a62338544dd/test_coverage)](https://codeclimate.com/github/DeveloperCAP/MLCAT/test_coverage)
+[![codecov](https://codecov.io/gh/DeveloperCAP/MLCAT/branch/master/graph/badge.svg)](https://codecov.io/gh/DeveloperCAP/MLCAT)
+[![Dependency Status](https://www.versioneye.com/user/projects/5a57a7d70fb24f382f102324/badge.svg?style=flat-square)](https://www.versioneye.com/user/projects/5a57a7d70fb24f382f102324)
 [![Requirements Status](https://requires.io/github/DeveloperCAP/MLCAT/requirements.svg?branch=master)](https://requires.io/github/DeveloperCAP/MLCAT/requirements/?branch=master)
```
Comment (prasadtalasila) [inline README.md]: @AakankshaSanctis please remove the requires.io badge. We have a versioneye badge which is serving the same purpose.

**Review by prasadtalasila — CHANGES_REQUESTED** [review]

**Comment by AakankshaSanctis** [comment]
README.md updated with removal of requirements.io badge

**Pull request merged**